app/modules/todo/models/todo_model.coffee

```diff
@@ -0,0 +1,38 @@
+Backbone = require 'backbone'
+
+class TodoModel extends Backbone.Model
+
+  # These are our default values (attributes) of our model
+  defaults:
+    'title': ''
+    'completed': false
+    'photo_path': null
+
+  # -------------------------------------------------------------------------- #
+  # Photo stuff
+  #
+  # Here we have some basic methods for checking if we have a photo and to set
+  # a photo
+  #
+  # Hooking the Cameara plugin to the model happens in the ItemView
+
+  # Do we have a photo?
+  has_photo: ->
+    photo_path = @get('photo_path')
+    if typeof(photo_path) == 'String' && photo_path != ''
```

Comment:
It's handy to know, it's good to make use of Underscore, for sure, but the reasoning why I use this kind of code, is, because it's vanilla JS, and that is already known by the audience taking the course. (At least we hope)